hedera-node/infrastructure-develop/dashboards/services-all.json

```diff
@@ -0,0 +1,258 @@
+{
```

Comment:
We should work with @beeradb to restructure the infrastructure folder to provide better organization and separation of production vs. non-production.

Reply:
@beeradb what do you think about it? Our goal is to have a place that devs can use to store dashboards.

Reply:
@beeradb any comments?